### DISCUSSION TITLE
Allow installing on Mac M1

### DISCUSSION THREAD

**Discussion by Freyja-Folkvangr** [community/community#3420] [Mobile]
Dear @github, the GitHub app works great on Mac with M1 chip. It would be much appreciated if you allow everyone to download through the App Store. (I’ve side installed it)

Screenshot:
https://twitter.com/_giosan/status/1364421142015062017?s=21

**Comment by lougreenwood** [comment]
I really want to see this happen too. I bought an M1 with the hope that I could get the iPad app as a dedicated interface for my GH workflow so that I could split it out from the sea of tabs in my browser!

(Sorry this is a "suggested answer", I don't see anyway to reply from the iOS app)

**Comment by shusso** [comment]
This would be excellent to have!

**Comment by ngeorger** [comment]
> Dear @github, the GitHub app works great on Mac with M1 chip. It would be much appreciated if you allow everyone to download through the App Store. (I’ve side installed it)
> 
> Screenshot:
> https://twitter.com/_giosan/status/1364421142015062017?s=21

In general, it would be nice if any app which has a technical user target (or at least a significant percentage), that feature comes as an almost obvious option. 
Anyways if some kind of specific device ux/ui experience is understandable, you would have even more kudos from us :) regards
